R/build-mock-url.R

```diff
@@ -79,9 +79,11 @@ get_string_request_body <- function(req) {
         if (inherits(x, "form_file")) {
           # hash the file contents
           paste("File:", digest(x$path, serialize = FALSE, file = TRUE))
-        } else {
-          # assume form_data
+        } else if (inherits(x, "form_data")){
           rawToChar(x$value)
+        } else {
+          # assume string
+          x
```

Comment:
This looks generally ok, but it looks like there's [another possibility in `req_body_multipart()`](https://github.com/r-lib/httr2/blob/main/R/req-body.R#L156): `# data must be character, raw, curl::form_file, or curl::form_data`

```suggestion
        } else if (is.raw(x) {
          rawToChar(x)
        } else {
          # assume string
          x
```

Reply:
Is it needed to transform raw data to character? If not, I would just change the comment to: 
```suggestion
        } else {
          # assume a character string or raw
          x
```

Reply:
Seems that rawToChar() is not needed